runtime/src/message_processor.rs

```diff
@@ -173,7 +173,10 @@ impl MessageProcessor {
                     compute_units_consumed,
                     result.is_err(),
                 );
-                timings.details.accumulate(&invoke_context.timings);
+                invoke_context.timings = {
```

Comment:
What is the scope for? Why not:
```Rust
timings.details.accumulate(&invoke_context.timings);
invoke_context.timings = ExecuteDetailsTimings::default();
```

Reply:
Just wanted to accumulate and reset in one expression. Not a fan?

Reply:
It is just that this assignment using a block scope - pattern is usually only used when it is needed, e.g. when there is a [RAII](https://en.wikipedia.org/wiki/Resource_acquisition_is_initialization) or borrow semantic.

Thus I was a bit confused, but it is fine either way.

Reply:
Ah got it, I like to use it for grouping related operations together as well. Thanks for raising